apps/chronicles/components/ComponentNavigationCell.tsx

```diff
@@ -24,6 +24,7 @@ export const ComponentNavigationCell = ({
     return (
         <Cell.Navigation
             title={title}
+            testID={title}
```

Comment:
Minor nitpick, should be renamed to `testId`

Reply:
I guess this property name was not defined in this pr so you can ignore this comment.

Reply:
The property name is default in react, similar to `ref` and `key`